server.js

```diff
@@ -67,11 +67,11 @@ app.use(sessions({
   },
 }));
 
-app.use("/", HomeRoutes);
 app.use("/", DockerflowRoutes);
 app.use("/scan", ScanRoutes);
 // app.use("/oauth", OAuthRoutes);
 // app.use("/user", UserRoutes);
+app.use("/", HomeRoutes);
```

Comment:
Personal curiosity, why did this line move? Was there a bug in the previous ordering?

Reply:
I added app.use("*", ... ) in the home routes. This needs to be added last, otherwise it will hijack everything else. Hence the move.